CMakeLists.txt

```diff
@@ -32,14 +32,16 @@ include_directories(
 find_package(Eigen3 REQUIRED)
 include_directories(${EIGEN3_INCLUDE_DIR})
 
+find_package(pybind11 REQUIRED)
+
```

Comment:
As we are doing a find here, we should remove the extra call that is being done in the `/python/CMakeLists.txt`